config/kubernetes-sigs/org.yaml

```diff
@@ -67,6 +67,7 @@ members:
 - mrunalp
 - msau42
 - munnerz
+- nikhita
```

Comment:
@cblecker is there any test to check if a name has been added in a non-alphabetical order?

I can't see anything like that from a quick glance in `config_test.go` and the verify scripts.

Reply:
No, I don't believe there is.

Reply:
This could be a unit test on config_test in this repo